tests/cypress/integration/actions_tasks2/case_30_collapse_sidebar_appearance.js

```diff
@@ -34,43 +34,13 @@ context('Collapse sidebar/appearance. Check issue 3250 (empty sidebar after resi
         cy.createRectangle(createRectangleShape2Points);
 
         // get default left value from background
-        cy.get('#cvat_canvas_background')
-            .should('have.css', 'left')
-            .then((currentValueLeftBackground) => {
-                defaultValueLeftBackground = Number(currentValueLeftBackground.match(/\d+/));
+        cy.get('.cvat-canvas-grid-root')
+            .then((el) => {
+                defaultValueLeftBackground = el[0].getBoundingClientRect().left;
             });
     });
 
     describe(`Testing case "${caseId}"`, () => {
-        it('Collapse sidebar. Cheeck issue 3250.', () => {
```

Comment:
Why that case is removed?

Reply:
Returned it partially, but in general I was not able to understand what is going on in this messy test, why it compares background coordinates and styles..